src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -578,6 +578,27 @@ void reorder_inputs::run(program& p, layout_optimizer& lo, reorder_factory& rf)
             }
         }
 
+        auto& input = conv_node.input();
+        if (conv_node.impl_type == impl_types::onednn && input.get_dependencies().front()->get_output_layout().format == format::bfyx) {
+            // For supporting optimizied onednn first conv, the input format from prev reorder to this conv is changed to a recommended format by onednn.
+            auto input_layout = input.get_output_layout();
+            if (lo.needs_onednn_bfyx_to_blocked(format::bfyx, conv_node.get_output_layout().format, input_layout, conv_node)) {
```

Comment:
Now we need to change the name of this function. It is not `bfyx_to_blocked` anymore.
(and I think we need to update the contents of the function itself, too)

Reply:
How about fixing them together at separate PR?